js-miniapp-sample/src/pages/event-listener.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { MiniAppEvents } from 'js-miniapp-sdk';
+
+import {
+  Button,
+  CardContent,
+  CardActions,
+  makeStyles,
+} from '@material-ui/core';
+
+import GreyCard from '../components/GreyCard';
+
+const useStyles = makeStyles((theme) => ({
+  scrollable: {
+    overflowY: 'auto',
+    width: '100%',
+    paddingTop: 20,
+    paddingBottom: 20,
+  },
+  card: {
+    width: '100%',
+    height: 'auto',
+  },
+  actions: {
+    justifyContent: 'center',
+  },
+  content: {
+    justifyContent: 'center',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    fontSize: 18,
+    color: theme.color.primary,
+    fontWeight: 'bold',
+    paddingBottom: 0,
+  },
+  info: {
+    fontSize: 16,
+    lineBreak: 'anywhere',
+    wordBreak: 'break-all',
+    color: theme.color.primary,
+    marginTop: 0,
+    paddingBottom: 10,
+  },
+}));
+
+let externalwebviewcloseeventcount = 0;
+let pauseeventcount = 0;
+let resumeeventcount = 0;
+
+window.addEventListener(MiniAppEvents.externalwebviewclose, function (e) {
+  let message = e.detail;
+  console.log(message);
+  externalwebviewcloseeventcount++;
+  const webviewclose = document.getElementById('webviewclose');
```

Comment:
See my other comment about reactively updating the variable. `getElementById` typically shouldn't need to be used in a React app.